.eslintrc.cjs

```diff
@@ -22,4 +22,16 @@ module.exports = {
     'object-shorthand': 'error',
     'prefer-arrow-callback': 'error',
   },
+  overrides: [
+    {
+      files: ['perf/*.js'],
+      env: {
+        browser: true,
+      },
+      rules: {
+        '@typescript-eslint/explicit-module-boundary-types': 'off',
+        '@typescript-eslint/ban-ts-comment': 'off',
```

Comment:
Maybe I'm misreading this?

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-ts-comment.md

But it seems like the default are all reasonable.

Reply:
There were 9 `@ts-ignore` comments in `perf/*.js` files that are now being flagged. The default from `ban-ts-comment` is that you may never use `@ts-ignore`, but you *can* use `@ts-expect-err` if you put a non-trivial comment after it. It turns out I could just remove all of these `@ts-ignore` lines and this exclusion, which I've now done.